### PR TITLE
add reference to react-testing-library

### PR DESCRIPTION
**Pull request by kentcdodds**
because it will help engineers write better tests with react. :)

### CODE REVIEW

File: content/docs/addons-test-utils.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ var ReactTestUtils = require('react-dom/test-utils'); // ES5 with npm
 > Note:
 >
 > Airbnb has released a testing utility called Enzyme, which makes it easy to assert, manipulate, and traverse your React Components' output. If you're deciding on a unit testing utility to use together with Jest, or any other test runner, it's worth checking out: [http://airbnb.io/enzyme/](http://airbnb.io/enzyme/)
+>
+> Alternatively, there is another testing utility called react-testing-library. This is a simple and small utility that helps your tests give you confidence your components are working by encouraging you to write tests that use your components as the end users use them. It also works with any test runner: [https://git.io/react-testing-library](https://git.io/react-testing-library)
```
Comment (alexkrolick) [inline content/docs/addons-test-utils.md]: ~~simple~~

~~small~~

Comment (kentcdodds) [reply content/docs/addons-test-utils.md]: Happy to make the change, but I'm curious what you have against the language. When compared to the previous recommendation of enzyme it is smaller and simpler so I thought it was a useful distinction to make.

Comment (alexkrolick) [reply content/docs/addons-test-utils.md]: IMO of course

- "Simple" is subjevtive and there's kind of a campaign to keep it out the docs (cc @acdlite) 
- Small could be more precise

> Alternatively, there is another testing utility called react-testing-library designed to encouraging writing tests that use your components as the end users use them.

Comment (kentcdodds) [reply content/docs/addons-test-utils.md]: I like your rewording. Mine was too wordy. I'll update. Thanks.
